utility.py

```diff
@@ -91,6 +91,9 @@ def sendMessage(subject, message):
                 server.sendmail(SENDER_EMAIL, RECEIVER_EMAIL,
                                 message.as_string())
                 sent = True
+        except smtplib.SMTPAuthenticationError:
+            print("Invalid email login information, please check the CONFIG file. Exiting...")
+            quit(1)
```

Comment:
Make sure to `import sys` at the top of the utility.py file and use `sys.exit()` instead of `quit()`. `sys.exit()` can take a string parameter so you could combine the `print("Invalid email login ...")` line with sys.exit(). 

```suggestion
            sys.exit("Invalid email login information, please check the CONFIG file. Exiting...")
```